src/Behavioral.Automation/Bindings/DropdownBinding.cs

```diff
@@ -65,12 +65,14 @@ public void CheckDropdownHeaders([NotNull] IDropdownWrapper wrapper, [NotNull] T
                 !behavior.Contains("not"), $"{wrapper.Caption} items are {items.Aggregate((x, y) => $"{x}, {y}")}");
         }
 
+        [Given("user selected \"(.*) in (.*)")]
         [When("user selects \"(.*)\" in (.*)")]
         public void ClickOnEntry([NotNull] string entry, [NotNull] IDropdownWrapper wrapper)
         {
             wrapper.Select(entry);
         }
 
+        [Given("user selected multiple entries in (.*)")]
```

Comment:
I think we need to use : symbol for consistency
`[Given("user selected multiple entries in (.*?):")]`

Reply:
Fixed